internal/data/kv_pair.go

```diff
@@ -560,6 +631,43 @@ func (rp kvPairRepo) RestoreCotaEntryKvPairs(ctx context.Context, blockNumber ui
 			return err
 		}
 
+		// sub key pair
+		{
```

Comment:
What is the purpose of the curly braces here?

Reply:
To group the logic so that the code is clearer. what do you think？

Reply:
I don't think it's necessary

Reply:
Ok, deleted